app/UiTiDv1/UiTiDv1ConsumerStatus.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UiTiDv1;
+
+enum UiTiDv1ConsumerStatus: string
+{
+    case Active = 'ACTIVE';
+    case Blocked = 'BLOCKED';
+    case Unknown = 'UNKNOWN';
+
+    public static function fromValue(string $name): self
+    {
+        foreach (self::cases() as $status) {
```

Comment:
You could use `tryFrom` and when that method returns `null` use `Unknown`.
See https://www.php.net/manual/en/backedenum.tryfrom.php

Reply:
cool, did not know that